frontend/public/components/secrets/create-secret.tsx

```diff
@@ -999,6 +1020,7 @@ type BaseEditSecretProps_ = {
   saveButtonText?: string;
   explanation: string;
   onCancel?: () => null;
```

Comment:
Check why this is done this way and change it null is not required.
```suggestion
  onCancel?: () => void;
```

Reply:
Done https://github.com/openshift/console/pull/1787/commits/916e8ed827da3840d79d3acfb4fb032335b67592